python/src/scipp/plot/controller1d.py

```diff
@@ -34,3 +34,22 @@ def toggle_xaxis_scale(self, owner):
         Toggle x-axis scale from toolbar button signal.
         """
         super().toggle_xaxis_scale(owner, normalize=True)
+
+    def rescale_to_data(self, button=None):
```

Comment:
Not clear to me why a new method was necessary to fix the bug? Can you elaborate?

Reply:
This was to fix the fact that limits were not being taken into account at all in 1d plots (which is also listed in the issue)

Reply:
Basically, in the way I implemented the fix for the 2d data, the `vmin` and `vmax` that get sent to `rescale_to_data` in `figure1d.py` will never be `None`.
It is much easier to let Matplotlib do the rescaling on 1d plots, especially when multiple lines are present on the same plot, as well as errorbars, instead of finding some min and max y limits manually.

Reply:
👍